src/app/components/super-admin-login/super-admin-login.component.css

```diff
@@ -12,37 +11,71 @@ div {
   justify-content: center;
   color: rgb(19, 20, 20);
 }
+.mat-mdc-card-header {
+  justify-content: center;
+}
+.mat-icon.mat-icon.notranslate.material-icons.mat-ligature-font.mat-icon-no-color {
```

Comment:
what is the need of this style

Reply:
.mat-mdc-card-header {
  justify-content: center;
}-----this, I used to make headre appear at the center as default styling to .mat-mdc-card-header is flex